tests/microos/workloads/x11-container/x11_helm_server.pm

```diff
@@ -0,0 +1,86 @@
+# SUSE"s openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Deploy X11, pulseaudio, firefox kiosk
+#          with Kubernetes Using Helm
+# Maintainer: Grace Wang <grace.wang@suse.com>
+
+use base 'consoletest';
+use warnings;
+use strict;
+use testapi;
+use lockapi;
+use mmapi;
+use utils;
+use mm_network 'setup_static_mm_network';
+use containers::k8s;
+use transactional;
+
+# MM network check: try to ping the gateway, the client and the internet
+sub ensure_client_reachable {
+    assert_script_run('ping -c 1 10.0.2.2');
+    assert_script_run('ping -c 1 10.0.2.102');
+    assert_script_run('curl conncheck.opensuse.org');
+}
+
+sub run {
+    my ($self) = @_;
+    my $release_name = "kiosk";
+    my $namespace = "kiosk";
+    my $helm_values = "kiosk_values.yaml";
+    my $helm_chart = "oci://registry.suse.de/suse/sle-15-sp6/update/bci/helmcharts/suse/kiosk/kiosk-chart";
+
+    select_console 'root-console';
+    set_hostname(get_var('HOSTNAME') // 'server');
+    setup_static_mm_network('10.0.2.101/24');
+    ensure_client_reachable();
+
+    # Permit ssh login as root
+    assert_script_run("echo 'PermitRootLogin yes' > /etc/ssh/sshd_config.d/root.conf");
+    assert_script_run("systemctl restart sshd");
+    assert_script_run "chmod a+rw /dev/snd/*";
+
+    # Install the package ca-certificates-suse and git
+    my $host_version = get_var("HOST_VERSION") ? 'HOST_VERSION' : 'VERSION';
+    my $distversion = 'SLE_' . get_required_var($host_version) =~ s/-SP/_SP/r;
+    enter_trup_shell;
+    zypper_call("ar http://download.suse.de/ibs/SUSE:/CA/$distversion/SUSE:CA.repo");
+    zypper_call("ar https://download.opensuse.org/distribution/leap-micro/6.1/product/repo/openSUSE-Leap-Micro-6.1-x86_64/ leap-micro-6.1");
+    script_run("zypper -n --gpg-auto-import-keys ref");
+    script_run("zypper install -y ca-certificates-suse git");
+    exit_trup_shell;
+    check_reboot_changes;
+
+    # Install k3s and helm
```

Comment:
You can assume that k3s and helm are installed on the host system, this is what @KuxaBeast is working on at the moment.

Reply:
The ticket was about pre-installing K3s, not Helm. However, I don't see an issue in adding it to the setup. https://progress.opensuse.org/issues/183983

Reply:
I am 100% sure it's available in `SL-Micro.x86_64-6.0.0-Default-Updated.qcow2`, it should also be available in the 6.1 variant.

Reply:
Just got the `SL-Micro.x86_64-6.1-Default-Updated.qcow2` and k3s is running without issues.

Reply:
Thanks for your confirmation.
I was using the wrong qcow. Will change from `SL-Micro.x86_64-6.1-Default-qcow-GM.qcow2` to `L-Micro.x86_64-6.1-Default-Updated.qcow2`.